src/components/lib/musl-1.1.11/src/internal/cos_syscalls.c

```diff
@@ -1,7 +1,7 @@
 __attribute__((weak, regparm(1))) long
 __cos_syscall(int syscall_num, long a, long b, long c, long d, long e, long f, long g)
 {
-        long *x = *(long*)0;
+        long *x = (long*)0;
```

Comment:
why not just `long *x = NULL`?

Reply:
Do we not use `NULL` because we need to avoid a dependency on `stdlib.h`?  Try it out and see.

Reply:
It doesn't cause any warnings or errors. But it is strange to think that stdlib.h can be included in musl src code.